package.json

```diff
@@ -83,7 +83,7 @@
   },
   "dependencies": {
     "classnames": "^2.2.0",
-    "cozy-ui": "3.0.0-beta39",
+    "cozy-ui": "^3.0.0-beta40",
```

Comment:
You shouldn't write the `^` before the version, it causes an issue when you update with yarn. Well, until the beta is here at least. Yarn can't handle version with text like beta in it. For it, it's still 3.0.0 regardless the beta number.